src/io-methods/google-cloud-storage.js

```diff
@@ -2,7 +2,7 @@ const { Storage } = require("@google-cloud/storage");
 
 const storage = new Storage();
 
-const regex = /^(?:google|gs):\/\/(.+)\/(.+)$/;
+const regex = /^(?:google|gs):\/\/(.+?)\/(.+)$/;
```

Comment:
See https://javascript.info/regexp-greedy-and-lazy

Reply:
that's a great article! thanks.